parser/parser.y

```diff
@@ -5221,6 +5230,53 @@ AdminStmt:
 			JobIDs: $6.([]int64),
 		}
 	}
+|	"ADMIN" "SHOW" "LOG" AdminShowLog
+	{
+		$$ = &ast.AdminStmt{
+			Tp: ast.AdminShowLog,
+			ShowLog: $4.(*ast.ShowLog),
+		}
+	}
+
+AdminShowLog:
+	"RECENT" NUM
+	{
+		$$ = &ast.ShowLog{
+			Tp: ast.ShowLogRecent,
+			Count: getUint64FromNUM($2),
+		}
+	}
+|	"TOP" NUM
+	{
+		$$ = &ast.ShowLog{
+			Tp: ast.ShowLogTop,
+			Count: getUint64FromNUM($2),
+		}
+	}
+|	"TOP" "USER" NUM
+	{
+		$$ = &ast.ShowLog{
+			Tp: ast.ShowLogTop,
+			Kind: "user",
+			Count: getUint64FromNUM($3),
+		}
+	}
+|	"TOP" "INTERNAL" NUM
```

Comment:
This syntax can be removed, use `ALL` is enough.

Reply:
Or Keep this and remove "ALL"

Reply:
How to represent the result "user" + "internal" ?
We have three cases:
* "user"
* "internal"
* "user" + "internal"

 one keyword is not enough